chain/ethereum/src/runtime/abi.rs

```diff
@@ -227,6 +300,79 @@ impl AscIndexId for AscEthereumEvent<AscEthereumTransaction_0_0_6, AscEthereumBl
     const INDEX_ASC_TYPE_ID: IndexForAscTypeId = IndexForAscTypeId::EthereumEvent;
 }
 
+#[repr(C)]
+#[derive(AscType)]
+pub(crate) struct AscEthereumLog {
+    pub address: AscPtr<AscH160>,
+    pub topics: AscPtr<AscTopicArray>,
+    pub data: AscPtr<Uint8Array>,
+    pub block_hash: AscPtr<AscH256>,
+    pub block_number: AscPtr<AscH256>,
+    pub transaction_hash: AscPtr<AscH256>,
+    pub transaction_index: AscPtr<AscBigInt>,
+    pub log_index: AscPtr<AscBigInt>,
+    pub transaction_log_index: AscPtr<AscBigInt>,
+    pub log_type: AscPtr<AscString>,
+    pub removed: AscPtr<AscWrapped<bool>>,
```

Comment:
Got a compiler error when trying to use a boolean directly:
```
the trait `AscIndexId` is not implemented for `bool`
```

Should we implement that trait for it?
I see we have an index ID for `WrappedBool`: 
https://github.com/graphprotocol/graph-node/blob/3841125d71522b69dd061ba740ac4310bda8ccc7/graph/src/runtime/mod.rs#L176

Reply:
Hmmm, have you tested sending a `false` and a `true`? The `WrappedBool` was for the case that we needed to pass a `Result<T, bool>` to the AS side.

I need to take a closer look if there's an alternative, can you double check that those work using [these tests](https://github.com/graphprotocol/graph-node/blob/master/runtime/test/README.md)? We had bugs in the past because of this.

Reply:
In this case, we decided that `AscWrapped<bool>` would be able to represent the original `Option<bool>` type signature for that field.